tests/dummy/app/components/menu-link-item.js

```diff
@@ -0,0 +1,15 @@
+import Ember from 'ember';
+import layout from '../templates/components/menu-link-item';
+
+const { Component } = Ember;
```

Comment:
I prefer using the module shims here: `import Component from 'ember-component';`

This approach will be more future-proof as we swap out the shimmed modules with actual npm modules.